### PR TITLE
Feat/bm components support

### PR DESCRIPTION
**Pull request by radireddy**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/baremetal-server-components
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-6936
| License          | BSD 3-Clause

## Description

Add reusable support tile component.

### CODE REVIEW

**Comment by frenautvh** [comment]
is it planned to refactor this part using this module ? https://github.com/ovh/manager/blob/develop/packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/technical-details/technical-details.controller.js

**Review by frenautvh — APPROVED** [review]

**Review by frenautvh — CHANGES_REQUESTED** [review]
mixing commits with https://github.com/ovh/manager/pull/4991

File: packages/manager/modules/bm-server-components/src/technical-details/controller.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+  formatRAM() {
+    const ram = get(this.technicalDetails, 'memory');
+    if (!ram) {
+      return '-';
+    }
+    const freqUnit = this.$translate.instant(
+      'dedicated_server_dashboard_technical_details_ram_frequency_unit',
+    );
+    const ramSize = ram.size ? `${ram.size} GB` : '';
```
Comment (varun257) [inline packages/manager/modules/bm-server-components/src/technical-details/controller.js:43]: `GB` should be translated here

File: packages/manager/modules/bm-server-components/src/technical-details/controller.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    return disks.map((disk) => {
+      const number = get(disk, 'number', 1);
+      const technology = get(disk, 'technology', '');
+      const diskInterface = get(disk, 'interface', '');
+      let capacity = Number(get(disk, 'capacity'));
+
+      if (Number.isNaN(capacity)) {
+        capacity = '-';
+      } else if (capacity >= 1000) {
+        capacity = `${Math.round((100 * capacity) / 1000, 2) / 100} TB`;
```
Comment (varun257) [inline packages/manager/modules/bm-server-components/src/technical-details/controller.js:85]: `TB` needs translation

File: packages/manager/modules/bm-server-components/src/technical-details/controller.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      } else {
+        capacity = `${capacity} GB`;
```
Comment (varun257) [inline packages/manager/modules/bm-server-components/src/technical-details/controller.js:87]: Same as above ?

**Review by ganeshkumar1989 — APPROVED** [review]

**Review by ganeshkumar1989 — CHANGES_REQUESTED** [review]
Seems to contain commits other than the intended ones...

**Pull request merged**